public/app/plugins/datasource/cloudwatch/__mocks__/dashboardOnLoadedEvent.ts

```diff
@@ -710,6 +710,30 @@ export const CloudWatchDashboardLoadedEvent = new DashboardLoadedEvent({
         sqlExpression: '',
         statistic: 'Average',
       },
+      {
+        alias: '',
+        datasource: {
+          type: 'cloudwatch',
+          uid: 'abc',
+        },
+        dimensions: {
+          InstanceId: '*',
+        },
+        expression: 'a / ',
+        hide: false,
+        id: '',
+        matchExact: true,
+        metricEditorMode: 0,
+        metricName: 'CPUUtilization',
+        metricQueryType: 0,
+        namespace: 'AWS/EC2',
+        period: '',
+        queryMode: '',
+        refId: 'B',
+        region: 'default',
+        sqlExpression: '',
+        statistic: '',
```

Comment:
In this query, statistic and metric is missing so it's incomplete. It should not be included in adoption tracking.